### DISCUSSION TITLE
can i check if my jobs are queued on hosted actions with a workflow, so i can start to run locally

### DISCUSSION THREAD

**Discussion by caiopfidalgo** [community/community#25046] [Actions]
<p>Hi,</p>
<p>What i want to know is if I can with a command in a workflow check the status of hosted queue, and if i have a queue in my hosted actions, my runner will go through self hosted action. So what I’m trying to say is, my workflow checks if i have something queued in my hosted actions and if is there a queue, it will start to run jobs through my self hosted actions</p>
<p>something like this:<br>
if: check_run[queued]<br>
runs-on: [self-hosted linux]</p>

**Comment by Yanjingzhu** [comment, marked as answer]
<p>Currently, there is no way to check hosted queue status. And we could not use if condition for runs-on keyword. </p>
<p>Hosted runner and self-hosted runner could not be the backup for each other.</p>
<p>You can execute up to 20 workflows concurrently per repository when using GitHub-hosted runners. This document shows detail usage limits: <a href="https://help.github.com/en/actions/automating-your-workflow-with-github-actions/about-github-actions#usage-limits" rel="nofollow noopener">https://help.github.com/en/actions/automating-your-workflow-with-github-actions/about-github-actions#usage-limits</a></p>
